src/components/spinner/spinner.js

```diff
@@ -0,0 +1,15 @@
+import './spinner.css';
+import spinnerContent from './spinner.hbs';
+import { createElementsFromString } from '../../common/utils';
+
+export default class Spinner {
+  static show(target) {
```

Comment:
How can I show several spinners at once?

Reply:
You can call Spinner.show(target) with different targets